cmd/bootstrapper/main.go

```diff
@@ -24,11 +26,13 @@ var (
 
 func init() {
 	rootCmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
-	logger.SetLogger(zap.New())
+	// Set up logger and send logs to Stdout
+	// Only info and warning level logs should be logged via logger. Errors should be redirected to Stderr instead
+	logger.SetLogger(zap.New(zap.WriteTo(os.Stdout)))
 }
 
 func main() {
 	if err := rootCmd.Execute(); err != nil {
-		log.Error(err, "wmcb execution failed")
+		os.Stderr.WriteString(fmt.Sprint(err, "wmcb execution failed"))
```

Comment:
Formatting is incorrect for Sprint()
Try: `fmt.Sprintf("wmcb execution failed: %s", err)`

Reply:
Updated